### ISSUE TITLE
Missing status of current search index

### ISSUE THREAD

**Issue by stv0g** [owncloud/search_lucene#58]
I'm missing a way to see the current status of the Lucene index.
Maybe a short status line in the configuration would be nice.

**Comment by karlitschek** [comment]
@butonic

**Comment by butonic** [comment]
@stv0g I want to move to a central index to allow implementing search in other files. Then this will be added to the admin page. A status in the user page also makes sense. Actually you may want to exactly know the indexing status for each of your files. That requires adding a message column to the lucene status table ... and I need an Idea how and when to show that in the UI. I was thinking of a status icon, maybe like :information_source: that reveals more info about a file. That might not only be used by search_lucene ... but could also show image dimensions and audio track length. @jancborchardt an opinion? Maybe a status sidebar on the right?

**Comment by jancborchardt** [comment]
If it’s still indexing, hovering over the search icon could show a tooltip saying `Indexing … 45 %`. When it’s done, nothing should be shown.

**Comment by stv0g** [comment]
Yeah, I noticed that. Unfortunately this popup only pops up for fraction of a second.
I think we could do better in terms of usability.

I like the solution @butonic proposed. There are  a lot more infos OC could provide for a file or directory.
Last accessed, sharing status, etc.. But thats something which has to be added to the core I guess.

**Comment by jancborchardt** [comment]
Let’s start with the most simple and non-invasive thing first. If we feel then that we need more, we can still do that.

The fact that we _could_ provide more info on something does not mean we _should_.

**Comment by butonic** [comment]
Ok, so where do I put info about indexing errors? On the users status page? Together with a reindex option? 

Am 4. Februar 2015 17:00:16 MEZ, schrieb Jan-Christoph Borchardt <notifications@github.com>:
>Let’s start with the most simple and non-invasive thing first. If we
>feel then that we need more, we can still do that.
>
>The fact that we _could_ provide more info on something does not mean
>we _should_.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/owncloud/search_lucene/issues/58#issuecomment-72880783

-- 
Diese Nachricht wurde von meinem Android-Mobiltelefon mit K-9 Mail gesendet.

**Comment by jancborchardt** [comment]
Just have the tooltip be permanent (to the left of the search icon) and say: »Indexing error, click to rescan«

**Comment by stv0g** [comment]
... and the current index status below the disk quota on the user page.
